devjokes/README.md

```
@@ -0,0 +1,14 @@
+# DevJokes
+
+## What are DevJokes?
+DevJokes are pages for developers that are publicly available on the TSC repo and site. These are for laughs or just not knowing what to work on but wanting to work on something.
+## How to commit in your designated folder
+1st, go to your folder and make a commit like a folder for sounds or images. Then, delete the placeholder file, called DeleteMe! or removeme.
+## Rules of DevJokes
+Please do not include any of the following:
+## Rules of DevJokes
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)